### PR TITLE
Added debug output for old and new indices for ES upgrade test

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### Description
Added debug output for old a new indices for the ES-4.4-4.5 upgrade test

/cc @periklis 
/assign @periklis @ewolinetz 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA:
- Enhancement proposal:

### CODE REVIEW

File: hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh [hunk 1]
```diff
@@ -213,16 +213,18 @@ log::info "Checking if the ES cluster is all yellow/green"
...
+#log::info "Reading new ES indices"
+#try_func_until_result_is_not_empty get_es_indices ${ES_POD_TIMEOUT}
+#new_indices=$(get_es_indices_names)
+
+#if [ "$old_indices" != "$new_indices" ]; then
+  #log::info "Test failed - Indices names not matched"
+  #log::info "Old indices: $old_indices"
+  #log::info "New indices: $new_indices"
+  #exit 1
+#fi
```
Comment (periklis) [inline hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh:225]: What is the intention here by commenting out? Testing or CI or permanently disable this checks? If latter applies I suggest to delete entirely.

**Review by periklis — CHANGES_REQUESTED** [review]

**Comment by ewolinetz** [comment]
this is for test only and only applies to release-4.5 -- manually adding valid bug label

File: hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh (hunk 1 above)
Comment (igor-karpukhin) [reply hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh:225]: I'd keep that so when we move this test to the golang we won't forget about verifying all indices

Comment (periklis) [reply hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh:225]: @igor-karpukhin Please ensure this happens also on master please.

**Review by periklis — COMMENTED** [review]
/lgtm

**Pull request merged**